### PR TITLE
Update docs README backporting instructions

### PR DESCRIPTION
**Pull request by laurapacilio**
Update the docs readme to make it explicit how to use backporting to update the latest docs version on the site. This is the most common use-case and we should be very explicit about it.
Screen Shot 2022-08-09 at 11 12 03 AM

### CODE REVIEW

File: website/README.md [hunk 1]
```diff
@@ -45,22 +45,28 @@ You should preview all of your changes locally before creating a pull request. T
...
 
-Backporting involves cherry-picking commits to one or more release branches within a docs repository. For example, if Terraform is currently on v1.2 and you need to add a security warning to the v1.1 documentation, you can backport (cherry-pick) commits to the v1.1 branch by labeling the PR with a backport label (e.g., 1.2-backport) associated with the release branch for the target version. 
+Backporting involves cherry-picking commits to one or more release branches within a docs repository. You can backport (cherry-pick) commits to a version branch by adding the associated backport label to your pull request. For example, if you need to add a security warning to the v1.1 documentation, you must add the `1.1-backport` label. When you merge a pull request with one or more backport labels, GitHub Actions opens a backport PR to cherry-pick your changes to the associated release branches. Someone needs to manually merge the backport PR to finish backporting the changes.
```
Comment (mgarrell777) [inline website/README.md:57]: Suggest changing "Someone" tin the final sentence o be a bit more specific as to who is able to manually merge the backport. Can literally anyone from anywhere do it? A member of the user's team? Or whom?

**Review by mgarrell777 — APPROVED** [review]
Hi @laurapacilio Thank you for updating these README instructions. I have one suggestion for clarity, but otherwise looks good to me.

File: website/README.md (hunk 1 above)
Comment (laurapacilio) [reply website/README.md]: fixed!! thank you!

**Review by mgarrell777 — APPROVED** [review]
Thanks again for your effort here. 1 small suggestion, but looks good otherwise.

File: website/README.md [hunk 2]
```diff
@@ -40,27 +45,33 @@ You should preview all of your changes locally before creating a pull request. T
...
+   <img width="317" alt="Screen Shot 2022-08-09 at 11 06 17 AM" src="https://user-images.githubusercontent.com/83350965/183686586-f94e58f3-fd62-48cf-88bd-fa886fe4724f.png">
+      
+1. Merge the pull request. GitHub Actions autogenerates a backport pull request, linked to the original.
+
+   <img width="726" alt="Screen Shot 2022-08-09 at 11 08 52 AM" src="https://user-images.githubusercontent.com/83350965/183687165-350b0e9b-a888-409e-91e2-81d82eac0a4e.png">
 
-When you merge a pull request with one or more backport labels, GitHub Actions opens a backport PR to cherry-pick your changes to the associated release branches. Someone needs to manually merge the backport PR to finish backporting the changes.  
 
-If the changes in the backport pull request are effectively equivalent to the original, you can review and merge your own backport pull request without waiting for another review. You can make minor adjustments to resolve merge conflicts, but you should not merge a backport PR that contains major content or functionality changes from the original, approved pull request.
+1. Merge the autogenerated backport pull request.
```
Comment (mgarrell777) [inline website/README.md:75]:
```suggestion
1. Merge the auto-generated backport pull request. 
```

Suggest hyphenating "autogenerated"

**Review by mgarrell777 — APPROVED** [review]

**Pull request merged**